src/main/java/fr/xephi/authme/command/executable/verification/VerificationCommand.java

```diff
@@ -0,0 +1,59 @@
+package fr.xephi.authme.command.executable.verification;
+
+import fr.xephi.authme.ConsoleLogger;
+import fr.xephi.authme.command.PlayerCommand;
+import fr.xephi.authme.data.VerificationCodeManager;
+import fr.xephi.authme.message.MessageKey;
+import fr.xephi.authme.service.CommonService;
+import org.bukkit.entity.Player;
+
+import javax.inject.Inject;
+import java.util.List;
+
+/**
+ * Used to complete the email verification process.
+ */
+public class VerificationCommand extends PlayerCommand {
+
+    @Inject
+    private CommonService commonService;
+
+    @Inject
+    private VerificationCodeManager codeManager;
+
+    @Override
+    public void runCommand(Player player, List<String> arguments) {
+        final String playerName = player.getName();
+
+        if (!codeManager.isEnabled()) {
+            ConsoleLogger.warning("Mail API is not set");
+            commonService.send(player, MessageKey.INCOMPLETE_EMAIL_SETTINGS);
```

Comment:
This is not necessarily true. I'll take care of this (dropping a comment so I don't forget)